src/main.rs

```diff
@@ -120,35 +133,39 @@ fn _main() -> IoResult<()> {
                     .help("sets cwd, must contain a Cargo.toml file"),
             )
             .arg(Arg::with_name("check").short("c").long("check").help(
-                "non-zero exit if Cargo.toml is unsorted, overrides default behavior",
+                "Returns non-zero exit code if Cargo.toml is unsorted, overrides default behavior",
             ))
             .arg(
                 Arg::with_name("print")
                     .short("p")
                     .long("print")
                     // No printing if we are running a --check
                     .conflicts_with("check")
-                    .help("prints Cargo.toml, lexically sorted, to stdout"),
+                    .help("Prints Cargo.toml, lexically sorted, to stdout"),
```

Comment:
I think conventionally help text is all lowercase unless there are multiple sentences. Do you mind changing back?

Reply:
I tried to be in line with the default texts for help and version which are
```
    -h, --help            Prints help information
    [...]
    -V, --version         Prints version information
```
but if you insist I can change back of course.

Reply:
Yeah, I guess stick with the capitalization, it seems like that is becoming more common.